src/components/AnchorForAttachmentsOnly/BaseAnchorForAttachmentsOnly.js

```diff
@@ -27,28 +30,13 @@ const defaultProps = {
 };
 
 class BaseAnchorForAttachmentsOnly extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            isDownloading: false,
-        };
-        this.processDownload = this.processDownload.bind(this);
-    }
-
-    /**
-     * Initiate file downloading and update downloading flags
-     *
-     * @param {String} href
-     * @param {String} fileName
-     */
-    processDownload(href, fileName) {
-        this.setState({isDownloading: true});
-        fileDownload(href, fileName).then(() => this.setState({isDownloading: false}));
-    }
-
     render() {
-        const source = addEncryptedAuthTokenToURL(this.props.source);
+        const sourceURL = this.props.source;
+        const sourceURLWithAuth = addEncryptedAuthTokenToURL(sourceURL);
+        const sourceID = (sourceURL.match(/chat-attachments\/(\d+)/) || [])[1];
```

Comment:
Let's add `/chat-attachments\/(\d+)/` to a CONST [here](https://github.com/Expensify/App/blob/b9e4fe95a75765472af5cccb68ba6520525e72a3/src/CONST.js#L743-L776)

Reply:
Yes, I'm aware of that, didn't seem harmful as we will get `undefined`. So the onyx key would we `download_undefined`. Do you have a better way to handle this?

Reply:
Added the regex.
@rushatgabhane If you have a better way to handle the unmatch case let me know, making this as resolved for now.